.vs/config/applicationhost.config

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="UTF-8"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file should be in the .gitignore.

Reply:
Strangely, the entire .vs folder *is* in the .gitignore. I suppose I can pull this file from the review anyways but it would be great if .gitignore would, you know, ignore.

Reply:
Related: Is there any reason we are not using the Visual Studio .gitignore that comes with a new GitHub repo? It's way more detailed.

Reply:
Because it wasn't generated when I created the project? Let's try the VS one and see if that solves the problem, since I've been seeing the same thing and I just keep having to stage changes around it.

Reply:
Oh, I see, this isn't a new check in, it's a change. So first we need to delete this file from the repo. That will likely just fix the problem.

Reply:
Yep, just added a commit to do that.